.babelrc

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
@anuragCES in current LTS version of Node.js (4.5.0) all ES6 features you use should be supported. There is no need for babel.